pkg/handler/testdata/rawPodHasAll.pod.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     testing.eks.amazonaws.com/serviceAccount/audience: "sts.amazonaws.com"
     testing.eks.amazonaws.com/handler/injectSTS: "true"
     testing.eks.amazonaws.com/handler/region: "cn-north-1"
+    eks.amazonaws.com/token-expiration: "86400"
```

Comment:
Can you add a second test that exercises the service account annotation? Also, can you use the non-default token expiration in a test?

Reply:
Done